### PR TITLE
magento/merchdocs#1154: Catalog| Product Workspace: Expand Product Scheduled Changes

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

Fix pull request fixes https://github.com/magento/merchdocs/issues/1154

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/catalog/product-scheduled-changes.html

whatsnew
New sections added to [Scheduled Changes](https://docs.magento.com/user-guide/catalog/product-scheduled-changes.html) to describe creating a new scheduled update, as well as assigning an existing update, viewing an update, and editing an update.

### CODE REVIEW

File: src/catalog/product-scheduled-changes.md [hunk 1]
```diff
@@ -3,7 +3,72 @@ ee_only: true
 title: Scheduled Changes
 ---
 
-Product updates can be applied on schedule and grouped with other content changes. You can create a new campaign based on scheduled changes to the product, or apply the changes to an existing campaign. For more information, see [Content Staging]({% link cms/content-staging.md %}).
+Product updates can be applied on schedule and grouped with other content changes. You can [content staging]({% link cms/content-staging.md %}), create a new campaign based on scheduled changes to the product, or apply the changes to an existing campaign.
```
Comment (leslietilling) [inline src/catalog/product-scheduled-changes.md:6]:
```suggestion
Product updates can be applied on schedule and grouped with other content changes. You can use [content staging]({% link cms/content-staging.md %}) to create a new campaign based on scheduled changes to the product, or apply the changes to an existing campaign.
```

File: src/catalog/product-scheduled-changes.md [hunk 2]
```diff
@@ -3,7 +3,72 @@ ee_only: true
...
 
 ![]({% link images/images-ee/product-scheduled-changes.png %}){: .zoom}
 _Scheduled Changes_
+
+## Create a New Schedule Update
```
Comment (leslietilling) [inline src/catalog/product-scheduled-changes.md:11]: Sentence case for headings
```suggestion
## Create a new schedule update
```

File: src/catalog/product-scheduled-changes.md [hunk 3]
```diff
@@ -3,7 +3,72 @@ ee_only: true
...
+
+1. Use the Calendar (![]({% link images/images/btn-calendar.png %})) tool to choose the **Start Date** and **End Date** for the campaign.
+
+1. Scroll down to _Price_ and click **Advanced Pricing**.
+
+1. Enter a **Special Price** for the product during the scheduled campaign and click <span class="btn">Done</span>.
+
+1. When complete, click <span class="btn">Save</span>.
+
+## Assign to Existing Update
```
Comment (leslietilling) [inline src/catalog/product-scheduled-changes.md:33]:
```suggestion
## Assign to existing update
```

**Review by leslietilling — COMMENTED** [review]
Edits for style

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling,thank you for reviewing!

I have implemented the suggestions you describe above. Please, check.

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1155`.

**Pull request merged**